app/admin/report.rb

```diff
@@ -12,4 +12,23 @@
     end
     f.actions
   end
+
+  show do
+    attributes_table do
+      row :name
+      row :query
+      row :shared
+      row :user
+      row :created_at
+      row :updated_at
+    end
+
+    panel "Scheduled Subscriptions" do
+      table_for report.subscriptions.order("created_at DESC") do |tbl|
+        tbl.column("Name") { |scheduled_report| link_to scheduled_report.name, admin_scheduled_report_path(scheduled_report) }
+        tbl.column("Owner") { |scheduled_report| scheduled_report.user }
```

Comment:
I think what code climate is suggesting is this:

`tbl.column("Owner")(&:user)`

but I do think what we have here is easier to read so fine with either